content/departments/engineering/dev/policies/vulnerability-management-policy.md

```diff
@@ -127,9 +127,9 @@ There are vulnerabilities that we confirm but may decide not to fix. For example
 
 Once accepted, a vulnerability will be considered closed but properly documented with the reasoning why the risk was accepted. Vulnerability acceptance needs approval from higher levels of management depending on their severity:
 
-- Critical: CTO
-- High: VP of Engineering
-- Medium: Engineering Director
+- Critical: Head of Product & Engineering
```

Comment:
i think Quinn was saying that he would hire separate people so these would be two different individuals, one for head of product and one for head of engineering. do we want to narrow this down to head of engineering?

Reply:
We will update when we have the new members here then.